src/services/eventService.ts

```
@@ -0,0 +1,45 @@
+    title: eventData.title,
+    description: eventData.description,
+    date: eventData.date,
+    location: eventData.location,
+    capacity: eventData.capacity,
+    price: eventData.price,
+    creatorId,
+    status: EventStatus.DRAFT,
+    id: isNumberObject(eventData.id) ? eventData.id : 0,
+    createdAt: new Date(),
+    updatedAt: new Date()
+
```

ESLint

[error] Replace `⏎` with `,`

(prettier/prettier)